extras/havpe-relay/init.py

```
@@ -0,0 +1,425 @@
+
+        wifi_ssid = self.prompt_value("WiFi SSID")
+        wifi_password = self.prompt_password("WiFi password")
+        # Auto-detect LAN IP as default
+        default_ip = ""
+        try:
+            s = socket.socket(socket.AF_INET, socket.SOCK_DGRAM)
+            s.connect(("8.8.8.8", 80))
+            default_ip = s.getsockname()[0]
+            s.close()
+        except Exception:
+            pass
```

Ruff (0.15.2)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)